



<launch>

  <node pkg="ar_week5_test" name="computer" type="compute_cubic_coeffs.py"/>

  <node pkg="ar_week5_test" name="generator" type="points_generator.py"/>

  <node pkg="ar_week5_test" name="planner" type="cubic_traj_planner.py"/>

  <node pkg="ar_week5_test" name="plotter" type="plot_cubic_traj.py"/>

</launch>
